Integrations/integration-Sandblast.yml

```diff
@@ -0,0 +1,249 @@
+        var res = http(
+                base + url,
+                {
+                    Method: 'POST',
+                    Body: body,
+                    Headers: headers
+                },
+                insecure
+            );
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Failed to ' + queryName + ' , request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
+        }
+        return JSON.parse(res.Body);
+    }
+
+    var teArgs = {'images': 0, 'reports': 0, 'benign_reports': 1};
+
+    var createRequest = function(args) {
+        if (!args.features) {
+            args.features = 'all';
+        }
+        if (args.features !== 'extraction' && args.features !== 'av') {
+            var feature = {};
+                var keys = Object.keys(teArgs);
+                for (var i = 0; i < keys.length; i++) {
+                    if (args[keys[i]]) {
+                        if (teArgs[keys[i]] === 0) {
+                            console.log(args[keys[i]])
+                            feature[keys[i]] = JSON.parse(args[keys[i]]);
+                        } else if (teArgs[keys[i]] === 1) {
+                            feature[keys[i]] = args[keys[i]] === 'true';
+                        }
+                        delete args[keys[i]];
+                    }
+                }
+                if (Object.keys(feature).length) {
+                    args['te'] = feature;
+                }
+        }
+        if (args.features === 'all') {
+            args.features = ['te', 'av', 'extraction'];
+        }
+        var request = {request: args};
+        return JSON.stringify(request);
+    }
+
+    switch (command) {
+        case 'test-module':
+            if (sendRequest(urlDict['sb-quota'], requestTemplate, contentTypeDict['sb-quota'], 'test')) {
+                return 'ok';
+            }
+            return 'not-cool';
+        case 'sb-download':
+            var res = http(
+                base + urlDict[command] + encodeToURLQuery(args),
+                {
+                    Method: 'GET',
+                    Headers: headers
+                },
+                insecure
+            );
+            if (res.StatusCode < 200 || res.StatusCode >= 300) {
+                throw 'Failed to ' + urlDict[command] + ' , request status code: ' + res.StatusCode + ' and Body: ' + res.Body + '.';
+            }
+            // Extract file name from response
+            var fileName = command + '_at_' + currentTime.getTime();
+            if (res.Headers && res.Headers['content-disposition'] && res.Headers['content-disposition'].length === 1) {
+                var matches = /'filename=\"(.*)\"'/.match(res.Headers['content-disposition'][0]);
+                if (matches) {
+                    fileName = matches[1];
+                }
+            }
+            return {Type: 9, FileID: saveFile(res.Body), File: fileName,  Contents: 'we must have contents for an entry'};
```

Comment:
Putting this content here, will make it load in a search.
Lets place something valid. like file name
BTW we do we have to have content ?